plugins/actix-web/src/lib.rs

```diff
@@ -279,6 +279,11 @@ where
         call(self, spec)
     }
 
+    // Returns the spec as a serde value. Does not continue to build the application.
+    pub fn spec(self) -> serde_json::Value {
+        serde_json::to_value(&*self.spec.read()).expect("generating json spec")
+    }
```

Comment:
Why can't we make use of `with_raw_json_spec` method? It passes the built JSON spec as `serde_json::Value`, which can be used for custom mounting and inspection, no?

Reply:
Yes, but unfortunately for situations like printing to stdout I still need to return something in that closure as I understand it: https://paperclip.waffles.space/paperclip/actix/struct.App.html#method.with_raw_json_spec -- this was the barrier I was running into. The result with this patch is now that my code looks something like this for my `dump-schema` subcommand:

```rust
pub fn dump_schema() -> std::io::Result<()> {
    std::io::stdout().lock().write_all(
        App::new()
            .wrap_api()
            .configure(configure)
            .spec()
            .to_string()
            .as_bytes(),
    )
}
```

Sorry if there's a better way of doing this and I'm just not aware of it.

Reply:
Maybe not looking that good but farily good for a CLI too, you could do it now like
```
use actix_web::test;
let mut app = test::init_service( || 
      App::new()
            .wrap_api()
            .configure(configure)
            .with_json_spec_at("/swagger")
            .build()
).await;
let schema: Value = test::read_response_json(&mut app, test::TestRequest::with_uri("/swagger"));
```

Reply:
Why can't I just open access to `spec` field of App object (with using getter) ?
In my opinion the only problem is that this is a private field.
I solved problem with a test query, but it's ... ugly!

Reply:
This is not a common requirement and has at least 2 ways to implement on a client side - via `with_raw_json_spec` and `test::init_service`.

Reply:
hey sorry for the incredibly late response. I've mostly moved past this problem in my project, not really willing to carry a torch for this patch.

If ya'll have a use for it, great, but if not, you won't upset me by closing it. Hope that's reasonable enough and nobody is upset.